ci-operator/step-registry/openshift/upgrade/qe/test/paused/openshift-upgrade-qe-test-paused-chain.yaml

```diff
@@ -0,0 +1,22 @@
+chain:
+  as: openshift-upgrade-qe-test-paused 
```

Comment:
This chain has exactly the same content as in `cucushift-upgrade-paused`, do we need to duplicate like this ?

Reply:
Our upgrade chain and arch migration chain are in the openshift/upgrade/qe/test, so moving EUS upgrade to the same folder to make them unified. Does it look good to you?

Reply:
Instead of copy, I'd prefer move.

Reply:
Removed cucushift-upgrade-paused chain and updated the tests which was using cucushift-upgrade-paused to openshift-upgrade-qe-test-paused